tasks/update-code/copy.yml

```diff
@@ -1,3 +1,6 @@
 ---
+- name: ANSISTRANO | SCP | Create release folder
```

Comment:
This is technically only required for the copy_unarchive strategy, because the tgz is copied into it before it is extracted. I am not sure if this is similar bebavior compared to the S3 strategy though.